src/index.ts

```diff
@@ -30,6 +32,27 @@ _addRunnerPlugin(() => {
   });
 });
 
-const test = baseTest.extend(fixtures);
+type JsonPrimitive = string | number | boolean | null;
+type JsonValue = JsonPrimitive | JsonObject | JsonArray;
+type JsonArray = JsonValue[];
+type JsonObject = { [Key in string]?: JsonValue };
```

Comment:
Can't we re-use these types as they are also used in `src/hooks.mts`

Reply:
Is this what you were thinking about?: [chore: reuse hooks config type](https://github.com/preactjs/playwright-ct/pull/6/commits/db68f970047c1cde1874964e1a8294bf2080bc5c)